Vagrantfile

```diff
@@ -87,6 +88,9 @@ Vagrant.configure(2) do |config|
     #git checkout <branch>
     mkdir -p /vagrant/images
     ln -s /vagrant/images images
-    sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --chroot'
+  SHELL
+
+  config.vm.provision "shell",  run: "always", inline: <<-SHELL
+    sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --noninteractive; sudo poweroff'
```

Comment:
better
`sudo -u vagrant screen -dmS build sudo bash -c 'export PATH="$PATH:/sbin:/usr/sbin";cd /home/vagrant/treehouse-builder;./treehouse-builder --noninteractive`

Reply:
Do we want to keep it on once treehouse-builder finishes?
what for?